frontend/src/metabase/query_builder/components/QuestionActions.styled.tsx

```diff
@@ -21,32 +13,3 @@ export const StrengthIndicator = styled(DatasetMetadataStrengthIndicator)`
   float: none;
   margin-left: 3.5rem;
 `;
-
-export type AnimationStates = "expand" | "shrink" | null;
```

Comment:
@npfitz Sorry to delete your implementation here, but since `BookmarkToggle` is already present, I remove this in favor of reusing a component instead. Let me know if there's any concern as I might not know all of the context why we need a separate bookmark toggle in the first place.